parts/k8s/kubeproxystart.ps1

```diff
@@ -24,4 +24,9 @@ Import-Module $global:HNSModule
 # and https://github.com/kubernetes/kubernetes/pull/78612 for <= 1.15
 Get-HnsPolicyList | Remove-HnsPolicyList
 
-.$KubeDir\kube-proxy.exe --v=3 --proxy-mode=kernelspace --hostname-override=$env:computername --kubeconfig=$KubeDir\config
+if (("--feature-gates=IPv6DualStack=true" | ? { $Global:ClusterConfiguration.Kubernetes.Kubelet.ConfigArgs -match $_ }) -ne $null) {
```

Comment:
Would it be possible to refactor this so that there is a single call to kube-proxy.exe at the end and conditionals are used when building an argument list?
That would make it easier to support other cases in the future and also more clear what is going on.

Reply:
+1. see parts/k8s/kubeletstart.ps1 for an example

Reply:
sure .will do that